app/send-team-notification.js

```diff
@@ -48,10 +48,10 @@ exports.sendTeamNotification = async function sendNotification(teamConfig, repor
     try {
 
         var notificationSent = false, errorMsg = "";
-        thresholdTime = "Threshold run at " + moment(moment().format()).format(util.dateFormat());
+        thresholdTime = teamConfig.isThresholdReport ? "Threshold run at " + moment(moment().format()).format(util.dateFormat()) : "Scheduled report run at " + moment(moment().format()).format(util.dateFormat());
```

Comment:
moment(moment().format()).format(util.dateFormat()) 
better to create in new function and place in utils folder if folder does not exist then create it